plugins/sample.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: sample
+spec:
+  homepage: https://github.com/seredot/kubectl-sample
+  shortDescription: Prints out sample manifest YAMLs
+  version: v1.0.0
+  description: |
+    Prints out a sample manifest YAML for a given resource type. 
+    The plugin works similar to `kubectl create deployment nginx --image=nginx --dry-run=client -o yaml` but for arbitrary resources.
```

Comment:
I'm not sure if there's a need to list that here + breaks the 80-char linewrap.

Reply:
It is now adjusted.